[
    {
        "content": "<p>Hello, we have different dataset, in which we upload .csv or .xlsx files in which from log comes out the error ‚Äú[2025-02-17T14:22:29.363-0500] [Payara 6.2024. 6] [INFO] [] [edu.harvard.iq.dataverse.util.FileUtil] [tid: _ThreadID=87 _ThreadName=http-thread-pool::jk-connector(2)] [timeMillis: 1739820149363] [levelValue: 800] [[<br>\n  FileName.csv is a file/extension name that Dataverse does not know. Consider adding it to the <a href=\"http://MimeTypeDetectionByFileName.properties\">MimeTypeDetectionByFileName.properties</a> file]]]<br>\n‚Äù, as this error can be solved. We also find that next to DRAFT VERSION, it generates a text ‚ÄúUNF:6:c1ibH+PvRoxetQ+3Yv7CLQ== [fileUNF]‚Äù</p>",
        "id": 500262675,
        "sender_full_name": "Andr√©s D√≠az Hurtado",
        "timestamp": 1739820938
    },
    {
        "content": "<p>Hello, any considered answer for this error?</p>",
        "id": 500424418,
        "sender_full_name": "Andr√©s D√≠az Hurtado",
        "timestamp": 1739891410
    },
    {
        "content": "<p>It's confusing. Are you able to upload these files to <a href=\"https://demo.dataverse.org\">https://demo.dataverse.org</a> so we can see how they look?</p>",
        "id": 501641173,
        "sender_full_name": "Philip Durbin üöÄ",
        "timestamp": 1740428760
    }
]